package.json

```diff
@@ -45,10 +45,10 @@
     "@babel/preset-react": "^7.14.5",
     "@babel/plugin-transform-react-jsx": "^7.14.9",
     "@material-ui/core": "^4.12.3",
-    "@nebula.js/cli": "1.7.0",
-    "@nebula.js/cli-build": "1.7.0",
-    "@nebula.js/cli-sense": "1.7.0",
-    "@nebula.js/cli-serve": "1.7.0",
+    "@nebula.js/cli": "2.0.0-beta.1",
+    "@nebula.js/cli-build": "2.0.0-beta.1",
+    "@nebula.js/cli-sense": "2.0.0-beta.1",
+    "@nebula.js/cli-serve": "2.0.0-beta.1",
```

Comment:
My idea is that we can create a release with this, so that it can easily be tested, by including this version of nebula and a table release on a branch in the client. Then we can make another release with nebula 2.0

Reply:
Okay, but that should probably be a beta release as well then

Reply:
sounds good